### ISSUE TITLE
[BUG]: cant get passed level 13 (mining quests)

### ISSUE THREAD

**Issue by amieex1** [The-Cove-Dev/wavy-mc#17]
### Discord Details

weirdamiee

### What happened?

level 13 states gold 2, iron 2, redstone 2 and lapis 1, ive done that and it wont let me get passed that level
![gold2](https://github.com/The-Cove-Dev/wavy-mc/assets/153443750/527cb748-82ca-48c6-9fae-8ec7fd250cf3)
![red2](https://github.com/The-Cove-Dev/wavy-mc/assets/153443750/15545795-11ff-4373-b769-b15e74a9e22f)
![gold1](https://github.com/The-Cove-Dev/wavy-mc/assets/153443750/1be375ed-b04c-49d3-8683-6260ab7e402b)
(ignore the fact it says gold1 and gold2 on the ss link)
![lapis1](https://github.com/The-Cove-Dev/wavy-mc/assets/153443750/63e3968d-3df0-4c27-9226-aff6734e4d02)

### Version

Java Edition (Most PC users)

**Comment by ImSnipper** [comment]
Did you complete the Deepslate quests, or the regular ore? 

I'm realizing now that the quests that are Deepslate do not explicitly state Deepslate - just show the deepslate ore in the GUI.

**Comment by JustMztt** [comment]
i noticed that awhile ago but just assumed it was intentional. does that mean none of the deepslate quests will actually say deepslate in the levels rn?

**Comment by amieex1** [comment]
yeah it doesnt matter if u complete deepslate or regular since i had no issue completing the last mining quest
